src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Skellige/Copper/DeafeningSiren.cs

```diff
@@ -22,5 +22,20 @@ public async Task HandleEvent(AfterWeatherApply @event)
             }
             return;
         }
+        public async Task HandleEvent(AfterTurnOver @event)
+        {
+            var count = 0;
+            if (@event.PlayerIndex != Card.PlayerIndex || !Card.Status.CardRow.IsInCemetery())
+                {return;}
+            if (Game.GameRowEffect[AnotherPlayer][RowPosition.EnemyRow1.Mirror().MyRowToIndex()].RowStatus == RowStatus.TorrentialRain)
+                {count += 1;}
+            if (Game.GameRowEffect[AnotherPlayer][RowPosition.EnemyRow2.Mirror().MyRowToIndex()].RowStatus == RowStatus.TorrentialRain)
+                {count += 1;}
+            if (Game.GameRowEffect[AnotherPlayer][RowPosition.EnemyRow3.Mirror().MyRowToIndex()].RowStatus == RowStatus.TorrentialRain)
+                {count += 1;}
+                {return;}
```

Comment:
Not sure why we have a return here?
```suggestion
                {count += 1;}
```

Reply:
Welll spotted mistake again, my bad, it's missing the line if (count == 0) before